blockboard/src/main/resources/static/js/event/postEvent.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * @author  Woohyeok Jun <woohyeok.jun@worksmobile.com>
+ * @file    postEvent.js
+ */
+
+ // 에디터 div 생성
+function editorAreaCreate() {
+  editorClear();
+  var editorcontent = $('#editorcontent');
+  var writecontent = $('#writecontent');
+  var btn_write = $('#btn_write');
+  writecontent.show();
+  writecontent.css("display", "");
+  btn_write.css("display", "none");
+  
+  $.template("editorTmpl",
+    '<h2> 게시글제목 </h2> ' +
+    '<input type="text" id="post_title" />' +
+    '<textarea id=editor></textarea>' + 
+    '<button id=btn_post>올리기</button>'
+  );
+  $.tmpl("editorTmpl").appendTo(editorcontent);
+  // textarea에 CKEditor 적용
+  $('#editor').ckeditor();
+}
+
+// 에디터 div 내용 삭제
+function editorClear() {
+  if(CKEDITOR.instances.editor) {
+    CKEDITOR.instances.editor.destroy();
+  }
+  $('#editorcontent').html("");
+  $('#writecontent').css("display", "none");
+  $('#btn_write').css("display", "");
+}
+
+// 게시글 조회 화면 Clear
+function postClear() {
+  $('#postcontent').html("");
+}
+
+// 게시글 목록 화면 Clear
+function postsClear() {
+  $('#postlist').html("");
+}
+
+// 현재 선택된 게시판 id 가져온다.
+function getCurrentBoardID() {
+  var tabs = $('#tab_id').children();
+  var boardID = 0;
+
+  $.each(tabs, function () {
+    var color = $(this).css('background-color');
+    if (color == "rgb(144, 238, 144)") {
+      boardID = $(this).attr('data-tab');
+    }
+  });
+
+  return boardID;
+}
+
+// '글쓰기' 버튼 이벤트
+$(document).on("click", "#btn_write", function () {
+  editorAreaCreate();
+});
+
+// '글쓰기' 버튼 클릭 후 '올리기' 버튼 이벤트
+$(document).on("click", "#btn_post", function () {
+  var postTitle = $('#post_title').val();
+  var postContent = CKEDITOR.instances.editor.getData();
+  var boardID = $('#boardIDinEditor option:selected').attr('data-tab');
+  if (postTitle == "") {
```

Comment:
다음과 같이 간략화가 가능하네요.
```suggestion
  if (postTitle) {
```
관련해서는 JS의 Falsy & Truthy 에 대해 조사해 보시면 좋을것 같네요.